ctlearn/image_mapping.py

```diff
@@ -787,11 +777,17 @@ def get_grids(self, pos, camera_type, grid_size_factor):
             if tick_diff % 2 != 0:
                 second_ticks.insert(0,np.around(second_ticks[0]-dist_second,decimals=3))
             
-            # Creating the virtual pixels outside of the camera.
-            virtual_pixel_x,virtual_pixel_y = self.add_virtualPixels(first_ticks, second_ticks, first_pos, second_pos, num_pixels)
-
-            first_pos=np.concatenate((first_pos,np.array(virtual_pixel_x)))
-            second_pos=np.concatenate((second_pos,np.array(virtual_pixel_y)))
+            # Create the virtual pixels outside of the camera
+            virtual_pixels = []
+            for i in [0, 1]:
+                j = i if camera_type != 'DigiCam' else 1 - i
+                virtual_pixels.append(self.get_virtual_pixels(
```

Comment:
I guess I know whats going on and causing the problem in the test_mapper notebook. It's a problem I faced a few times and caused by the default padding, which can be an even or odd number for different hexagonal conversion methods. This is why it's occurred irregular. If we want to (default) pad an odd number, the grid is shifted by one row. This implies adding virtual pixels between the cameras pixels (blue = camera pixels; red = virtual pixels; green output gridpoints):
![lstcam_virtualpixels](https://user-images.githubusercontent.com/37835610/52631897-58355700-2ec0-11e9-9623-19a2affef058.png)

I will add an extra 'if statement' to check if default_padding is even or odd and add LSTCam and NectarCam to DigiCam, since they have to treat equally. I think that should fix the problem.